src/Post_Command.php

```diff
@@ -171,7 +178,28 @@ public function create( $args, $assoc_args ) {
 		}
 
 		$array_arguments = array( 'meta_input' );
-		$assoc_args      = \WP_CLI\Utils\parse_shell_arrays( $assoc_args, $array_arguments );
+		$assoc_args = \WP_CLI\Utils\parse_shell_arrays($assoc_args, $array_arguments);
```

Comment:
This entire block (lines 181-202) seems to have been inadvertently reformatted to PSR2 with the last commit.

Please adapt the formatting to adhere to WPCS instead.